primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.selectcheckboxmenu.js

```diff
@@ -625,7 +625,12 @@ PrimeFaces.widget.SelectCheckboxMenu = PrimeFaces.widget.BaseWidget.extend({
 
             var togglerCheckboxInput = this.toggler.find('> div.ui-helper-hidden-accessible > input');
             this.bindCheckboxKeyEvents(togglerCheckboxInput);
-            togglerCheckboxInput.on('keyup.selectCheckboxMenu', function(e) {
+            togglerCheckboxInput.on('keydown.selectCheckboxMenu', function(e) {
+                if(e.which === $.ui.keyCode.TAB && e.shiftKey) {
+                    e.preventDefault();
+                }
+            })
+                .on('keyup.selectCheckboxMenu', function(e) {
                         if(e.which === $.ui.keyCode.SPACE) {
```

Comment:
Just a minor styling thing here can you make the next .on line up with the paren above it so they look chained.  You can see other compoennts for examples.

Reply:
has been added